### PR TITLE
added 3 new workspace nodepools for preview-environments

### PR DESCRIPTION
**Pull request by wulfthimm**
## Description

This adds new nodepools for workspaces. [This PR](https://github.com/gitpod-io/ops/pull/801) needs to be applied before this is merged.

## Related Issue(s)

Fixes #https://github.com/gitpod-io/ops/issues/800

## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **gtsiolis** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @gtsiolis` in a comment when ready.

Associated issue: *#800*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mads-hartmann — APPROVED** [review]
Thanks, LGTM. Added the hold label so it won't get merged automatically. Just remove it when you're ready to merge.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 0a839972d8e0fa5b4e636bfb22a7d0292d4b58c6</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7755#pullrequestreview-859562943" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **gtsiolis** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @gtsiolis` in a comment when ready.

Associated issue: *#800*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by wulfthimm** [comment]
https://github.com/gitpod-io/ops/pull/801 has been merged.

**Pull request merged**
